### PR TITLE
Adding vallox number platform documentation

### PR DESCRIPTION
**Pull request by yozik04**
## Proposed change

Adding documentation about number platform.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

I am not sure if I should notify about service deprecation in the documentation or just remove the whole block.

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/75858
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/vallox.markdown [hunk 1]
```diff
@@ -40,23 +43,27 @@ The four standard Vallox profiles are supported:
...
+### Service `vallox.set_profile_fan_speed_home` (deprecated)
 
 Set the fan speed of the `Home` profile.
 
 | Service data attribute | Optional | Description                                     |
 |------------------------|----------|-------------------------------------------------|
 | `fan_speed`            |       no | Fan speed in %. `Integer`, between 0 and 100.   |
 
-### Service `vallox.set_profile_fan_speed_away`
+### Service `vallox.set_profile_fan_speed_away` (deprecated)
```
Comment (frenck) [inline source/_integrations/vallox.markdown]: Please remove deprecated services from the documentation.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — DISMISSED** [review]

**Comment by yozik04** [comment]
I am sorry @frenck. But main PR review showed that I need to move some number platform entities to fan platform. Fan platform related PR will deprecate mentioned services. Not current one.

**Review by MartinHjelmare — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @yozik04 👍

**Pull request merged**